feature-configs/demo/sriov/12-sriov-network.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: sriovnetwork.openshift.io/v1
```

Comment:
I would put this to base (and eventually kustomize the ipam section). 
Do we have dhcp in the demo environment?